python/pyarrow/_fs.pyx

```diff
@@ -100,6 +100,53 @@ cdef class FileInfo(_Weakrefable):
         If given, the filesystem entry size in bytes.  This should only
         be given if `type` is `FileType.File`.
 
+    Examples
+    --------
+    Generate a file:
+
+    >>> from pyarrow import fs
+    >>> local = fs.LocalFileSystem()
+    >>> with local.open_output_stream('/tmp/fileinfo.dat') as stream:
+    ...     stream.write(b'data')
+    ...
+    4
```

Comment:
This "4" is a left-over? (or at least for me, it doesn't print the size of what gets written)

Reply:
Hm, it prints it out for me and also doctest is happy about it. Will try to run doctest without the "4".

Reply:
Yes, doctest complains if I remove the "4" ...

Reply:
Oh, I guess if I remove extra '...' then doctest doesn't like the numbered output ("4") also. So I will remove '...' and the extra numbers 👍

Reply:
That's bizarre. So in an IPython console, I don't see anything printed, but in a plain python console I indeed see the same ...